### ISSUE TITLE
Error for Apnonce reading unjailbroken Iphone/ipad

### ISSUE THREAD

**Issue by Zedris** [airsquared/blobsaver#313]
Your issue will be closed and/or deleted if you do not fill out this template.

Please remove the following line and the text above it before submitting this issue:

-------------------------------------------------------------

**Describe the bug**
Reading Apnonce for an unjailbroken iphone and ipad pro 11inch ver 3  is providing an error with the log provided below. 

**Desktop:**
 - OS Version:Beta 3 ios 15
 - iOS Device:iPhone XR/Ipad Pro 3 11 inch

**Checklist (place an x between the brackets to mark as completed):**
 - [x] I have made sure I am on the latest version of blobsaver
 - [x] I have checked the blobsaver wiki to see if there is already a solution to my problem
 - [x] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
If applicable, paste any log(s) here, on the lines between the backticks.
```
iphone:
java.lang.NullPointerException: Cannot invoke "com.sun.jna.Pointer.getByteArray(long, int)" because the return value of "airsquared.blobsaver.app.natives.Libplist.getDataPtr(com.sun.jna.Pointer, com.sun.jna.ptr.IntByReference)" is null
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.plistDataToString(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.getApNonceNormalMode(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

ipad: 

java.lang.NullPointerException: Cannot invoke "com.sun.jna.Pointer.getByteArray(long, int)" because the return value of "airsquared.blobsaver.app.natives.Libplist.getDataPtr(com.sun.jna.Pointer, com.sun.jna.ptr.IntByReference)" is null
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.plistDataToString(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.getApNonceNormalMode(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

**Comment by K4CZP3R** [comment]
#312 exact the same thing here

**Comment by airsquared** [comment]
This has been fixed in v3.0.2.
